data/110/869/629/7/1108696297.geojson

```diff
@@ -6,16 +6,19 @@
     "edtf:inception":"uuuu",
     "geom:area":1.245526,
     "geom:area_square_m":14491424023.555618,
-    "geom:bbox":"79.7438089689,18.6800200812,80.8995575944,20.8314971393",
+    "geom:bbox":"79.74380896888859,18.68002008116867,80.89955759435033,20.83149713932237",
     "geom:latitude":19.784918,
     "geom:longitude":80.241083,
     "iso:country":"IN",
     "label:eng_x_preferred_longname":[
-        "Garhchiroli District"
+        "Gadchiroli District"
     ],
     "label:eng_x_preferred_placetype":[
         "district"
     ],
+    "label:eng_x_variant_longname":[
```

Comment:
Nit, we probably don't need to maintain variants for labels / longnames like we do names, that's beyond the scope of our versioning documentation and can just bloat things up.

It's not wrong, so OK to merge as-is and just watch it going forward. Or clean this PR up :)

Reply:
Hmm.. I tried to fix this in a new commit, but hit some issues because this is a PR from write-field. I will note this going forward, but merge this change as-is. 👍